python/ray/tests/test_object_spilling.py

```diff
@@ -111,6 +119,68 @@ def test_url_generation_and_parse():
     assert parsed_result.size == size
 
 
+@pytest.mark.skipif(
+    platform.system() == "Windows", reason="Failing on Windows.")
+def test_default_config(shutdown_only):
+    ray.init(num_cpus=0, object_store_memory=75 * 1024 * 1024)
+    # Make sure the object spilling configuration is properly set.
+    config = json.loads(
+        ray.worker._global_node._config["object_spilling_config"])
+    assert config["type"] == "filesystem"
+    assert (config["params"]["directory_path"] ==
+            ray.worker._global_node._session_dir)
+    # Make sure the basic workload can succeed.
+    run_basic_workload()
+    ray.shutdown()
+
+    # Make sure config is not initalized if spilling is not enabled..
+    ray.init(
+        num_cpus=0,
+        object_store_memory=75 * 1024 * 1024,
+        _system_config={
+            "automatic_object_spilling_enabled": False,
+            "object_store_full_delay_ms": 100
```

Comment:
Btw, I just kept the config of other tests because it was a little tricky to make the general fixture that works in every case.

Reply:
(There were lots of issues I went through haha)